app/models/concerns/deprecated_group_features.rb

```diff
@@ -0,0 +1,34 @@
+require 'active_support/concern'
+
+#### NB: These methods are deprecated and should not be relied upon going forward
+module DeprecatedGroupFeatures
```

Comment:
What do you think of this pattern, @robguthrie? I've had the urge a couple times to go through and clear out a bunch of junk from the models and put them in things like this, just to simplify them a little and identify what we're using and what we're not.